### PR TITLE
MA3 and Atmospherics Adjustments

### PR DESCRIPTION
**Pull request by Desolane**
Read commits

### CODE REVIEW

**Review by bloxgate — APPROVED** [review]
Looks good as far as I can tell from just reading it. I'd appreciate it if someone that had some more time tonight did a test locally.

I'm a bit weary about removing pipenet purging for when we ever get some semblance of an atmospherics setup on one of the ships, but I suppose we can add that back if and when its needed.

**Comment by Desolane** [comment]
I only commented out pipeline purging for just such a reason

**Comment by caelaislinn** [comment]
We shouldn't be commenting out code unless there is a clear reason, particularly debugging or admin tools. Many of our maps have some kind of piped atmospherics and right now I can't see an issue with keeping that code.

File: code/modules/admin/verbs/grief_fixers.dm [hunk 1]
```diff
@@ -17,15 +17,15 @@
...
+/*
 	// Remove all gases from all pipenets
 	for(var/datum/pipe_network/PN in pipe_networks)
 		for(var/datum/gas_mixture/G in PN.gases)
 			G.gas = list()
 			G.update_values()
 
 	to_chat(usr, "\[2/5\] - All pipenets purged of gas.")
-
+*/
```
Comment (caelaislinn) [inline code/modules/admin/verbs/grief_fixers.dm]: Why is this commented out?

**Review by caelaislinn — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/grief_fixers.dm (hunk 1 above)
Comment (Desolane) [reply code/modules/admin/verbs/grief_fixers.dm]: It stops the admin verb of fixing atmos grief from purging all fuel in a ship's engine storage.

Comment (caelaislinn) [reply code/modules/admin/verbs/grief_fixers.dm]: The point of this verb is to clear ALL atmos on the server including pipenets, so the code should be left. If you want to add a separate verb which only wipes ZAS zones then that would be fine.

Comment (Desolane) [reply code/modules/admin/verbs/grief_fixers.dm]: Will do a new verb with zone clearing and leave this verb untouched then.

**Review by caelaislinn — APPROVED** [review]

**Comment by caelaislinn** [comment]
Approved pending travis

**Comment by Desolane** [comment]
This is ready to merge.

**Comment by caelaislinn** [comment]
Please have a description for PRs

**Comment by Desolane** [comment]
Added description.

**Comment by caelaislinn** [comment]
The PR should summarise the commits...

**Pull request merged**